### ISSUE TITLE
[Bug] Validating with two clicks a payment adds two payments

### ISSUE THREAD

**Issue by mdeweerd** [Dolibarr/dolibarr#7721]
# Bug
Two payments generated from the same page were added after clicing two times on the validation button.
As a result, the invoice was considered "payed" but the end amount was not 0.

## Environment
- **Version**: 6.0.2

## Expected and actual behavior
Only 1 transaction should be added when validating a payment.
Further, when the invoice is paid, it is not possible to add a new transaction from the user interface.  The verification of this limitation should be done in the model also (when actually applying the transaction).

**Comment by atm-maxime** [comment]
Sorry I can't reproduct in a 6.0.4. I tried on customer invoice and supplier invoice, but when I click multiple times on "Validate" in the confirmation box, I only have 1 payment.
Can you reproduce in 6.0.4 and give us screnn shots ?

**Comment by mdeweerd** [comment]
I am sorry, but 6.0.4 is not released yet, and it is not on demo.org either.

However, I tried in 6.0.3 and on demo.org which is in 6.0.2.

On the demo site I have chosen a random invoice and I continued clicking "Validate" for some time.
The result is this (https://demo.dolibarr.org/compta/facture/card.php?facid=5599 ):

![image](https://user-images.githubusercontent.com/1504752/32971600-2f198b42-cbee-11e7-9d5a-b470b971b63a.png)
